src/db.test.ts

```diff
@@ -0,0 +1,284 @@
+			const column = bytes('data_bytes');
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+
+		it('should work with Bytes instances', () => {
+			const testBytes = Bytes.from('0x1234567890abcdef');
+			expect(testBytes).toBeInstanceOf(Bytes);
+			expect(testBytes.data).toBeInstanceOf(Buffer);
+		});
+	});
+
+	describe('bytes32', () => {
+		it('should be a function', () => {
+			expect(typeof bytes32).toBe('function');
+		});
+
+		it('should create a column when called', () => {
+			const column = bytes32();
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+
+		it('should create a column with a name when called with name', () => {
+			const column = bytes32('hash_bytes');
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+	});
+
+	describe('uint256', () => {
+		it('should be a function', () => {
+			expect(typeof uint256).toBe('function');
+		});
+
+		it('should create a column when called', () => {
+			const column = uint256();
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+
+		it('should create a column with a name when called with name', () => {
+			const column = uint256('balance');
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+
+		it('should work with Uint instances', () => {
+			const testUint = new Uint(BigInt('12345678901234567890'));
+			expect(testUint).toBeInstanceOf(Uint);
+			expect(testUint.value).toBe(BigInt('12345678901234567890'));
+		});
+	});
+
+	describe('int256', () => {
+		it('should be a function', () => {
+			expect(typeof int256).toBe('function');
+		});
+
+		it('should create a column when called', () => {
+			const column = int256();
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+
+		it('should create a column with a name when called with name', () => {
+			const column = int256('signed_value');
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+
+		it('should work with Int instances', () => {
+			const testInt = new Int(BigInt('-12345678901234567890'));
+			expect(testInt).toBeInstanceOf(Int);
+			expect(testInt.value).toBe(BigInt('-12345678901234567890'));
+		});
+	});
+
+	describe('struct', () => {
+		it('should be a function', () => {
+			expect(typeof struct).toBe('function');
+		});
+
+		it('should create a column when called', () => {
+			const column = struct<{ name: string; age: number }>('test_struct');
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+
+		it('should create a column with a name when called with name', () => {
+			const column = struct<{ name: string; age: number }>('user_data');
+			expect(column).toBeDefined();
+			expect(typeof column).toBe('object');
+		});
+	});
+
+	describe('Address integration', () => {
+		it('should work with Address.from() method', () => {
```

Comment:
Can we add negative tests as well (i.e., invalid length, invalid characters)

Reply:
removed tests